content/issues/tracking-your-work-with-issues/deleting-an-issue.md

```diff
@@ -14,13 +14,13 @@ versions:
 topics:
   - Pull requests
 ---
-The ability to delete issues depends on whether the repository is owned by your personal account or an organization:
-- You can only delete issues in a repository owned by your personal account.
-- You cannot delete issues in a repository owned by an organization or another personal account, even if you are a collaborator there.
+The ability to delete issues depends on whether the repository is owned by a personal account or an organization:
+- The only account that can delete issues in a personal account repository is that account.
```

Comment:
nit suggestion to avoid stacked modifiers:
```suggestion
- The only account that can delete issues in a repository owned by a personal account is that account.
```